app/presenters/finder_presenter.rb

```diff
@@ -218,7 +218,7 @@ def all_content_finder?
   # FIXME: This should be removed once we have a way to determine
   # whether to display metadata in the finder definition
   def eu_exit_finder?
-    self.content_item['content_id'] == "42ce66de-04f3-4192-bf31-8394538e0734"
+    EuExitFinderHelper.eu_exit_finder? self.content_item['content_id']
```

Comment:
Do we still need this method? Could we just replace all instances of [`eu_exit_finder?`](https://github.com/alphagov/finder-frontend/search?q=eu_exit_finder%3F&unscoped_q=eu_exit_finder%3F) with a call to the new module?